src/main/java/net/rptools/maptool/client/ui/htmlframe/HTMLOverlayPanel.java

```diff
@@ -117,6 +119,20 @@ public ConcurrentSkipListSet<HTMLOverlayManager> getOverlays() {
     return overlays.clone();
   }
 
+  public HTMLOverlayManager init() {
```

Comment:
I think this approach to fixing #4954 actually masks the issue rather than fixing it. Did some testing, I believe the only thing that needs to change is in `showOverlay()` - we need `needsSorting` to be true even for internal overlays. Otherwise the new overlay ends up in front of the `front` region.

Also I have a personal motivation to not have overlays automatically open :slightly_smiling_face: - see #5044. Overlays on my system cause a lot of input lag, and this would force me to endure it despite not using the stat sheet feature yet.

Reply:
You're correct on both points (masking,sorting) and I looked right past that, thank you sir!